echopype/convert/set_groups_base.py

```diff
@@ -73,8 +73,8 @@ def set_toplevel(self, sonar_model, date_created=None) -> xr.Dataset:
 
     def set_provenance(self) -> xr.Dataset:
         """Set the Provenance group."""
-        prov_dict = echopype_prov_attrs(process_type="conversion", source_files=self.input_file)
-        ds = xr.Dataset()
+        prov_dict = echopype_prov_attrs(process_type="conversion")
+        ds = xr.Dataset(source_files_vars(self.input_file))
```

Comment:
I see that previously we only stored the `raw_file` path from `open_raw` as an attribute. However, for AZFP we have both the `raw_file` and `xml_path` as inputs and the file names for these can be quite different. Should we make it so that we store both the `raw_file` and `xml_path` in the variable?

Reply:
Good point. I agree that storing/preserving `xml_path` would be a good idea, or at least something we should discuss at some point. But I think that's a new development that would be outside the scope of this PR. This PR doesn't change *what* is stored, just *how* it is stored.

Reply:
I agree that it is outside the scope of this PR. If we do need to implement it, I think that would be a breaking change. So, we would need to fit this into v0.6.0, right?

Reply:
As a new feature, I would push this to >0.6.0. As usual: new issue!